app/partials/app.cjsx

```diff
@@ -19,9 +19,12 @@ PanoptesApp = React.createClass
     geordi: @geordiLogger
 
   getEnv: ->
+    shell_env = process.env.NODE_ENV
     reg = /\W?env=(\w+)/
     browser_env = window?.location?.search?.match(reg)
-    @state?.env || browser_env?[1] || 'staging'
+    env = shell_env || browser_env?[1] || 'staging'
```

Comment:
I do kind of like the `shell_env` but then I'd suggest moving the `?[1]` away from this line, so that you just get `shell_env || browser_env || 'staging'`. Kind of weird if `shell_env` contains a string, but `browser_env` is actually a regex match result.